### PR TITLE
Add jbrowse 1.12.1 recipe

### PR DESCRIPTION
**Pull request by abretaud**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

@bioconda/core
A recipe for jbrowse, will be used for galaxy tools in IUC

### CODE REVIEW

**Comment by abretaud** [comment]
Green now \o/

File: recipes/perl-heap-simple-perl/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  fn: Heap-Simple-Perl-0.14.tar.gz
+  url: http://cpan.metacpan.org/authors/id/T/TH/THOSPEL/Heap-Simple-Perl-0.14.tar.gz
+  md5: a2723ab28b7fcb35131037525a5b2f93
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl
```
Comment (abretaud) [inline recipes/perl-heap-simple-perl/meta.yaml]: Done!

**Comment by abretaud** [comment]
Hej,
Any objection against merging this?

**Comment by bgruening** [comment]
Thanks!

**Comment by johanneskoester** [comment]
The package perl-text-tabs+wrap apparently can't be uploaded by anaconda (see [here](https://travis-ci.org/bioconda/bioconda-recipes/jobs/170697949#L504)). I guess the reason might be the unusual naming? Also, when I try to access the given homepage of the package(http://metacpan.org/pod/Text::Tabs+Wrap), cpan can't find it. Are you sure about the name?

**Comment by abretaud** [comment]
Yep, it was the problem of naming, I just fixed it in #2705

**Pull request merged**